src/patternfly/components/Page/page.scss

```diff
@@ -74,7 +74,11 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
   --pf-c-page__sidebar--Width: #{pf-size-prem(290px)};
   --pf-c-page__sidebar--BackgroundColor: var(--pf-global--BackgroundColor--dark-300);
   --pf-c-page__sidebar--m-light--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
-  --pf-c-page__sidebar--BoxShadow: var(--pf-global--BoxShadow--lg-right);
+  --pf-c-page__sidebar--BoxShadow: none;
```

Comment:
I think want to keep this for < `$pf-global--breakpoint--xl`. You lost your `box-shadow` when the sidebar appears above the page content.

![Screen Shot 2022-07-07 at 1 29 46 PM](https://user-images.githubusercontent.com/5385435/177834634-b9aad4e2-1ccb-4cf2-ab5c-10f79d4aea52.png)